### ISSUE TITLE
W2 Ch7

### ISSUE THREAD

**Issue by JSchuurmans** [JannesKlaas/MLiFC#9]
from keras.models import Sequential
Using TensorFlow backend
'Couldn't import dot_parser, loading of dot files will not be possible.'

'...but there are other possibilities, too.' What is the comma doing there?

...optimizers with different hyper parameters. Maybe explain (or comment in code) that sgd is stochastic gradient decent.

GPUs where originally invented to render computer game graphics. -> render computer graphics. (Visual artists and engineers need it as well for movie rendering and 3d models.)

To see how good our model actually is or weather it overfits the training set, let's evaluate on the test set. (I believe training set is mentioned once before in week 1. There could be more emphasis on training and testsets, crossvalidation, datasnooping and overfitting. As these are one of the most important concepts in ML).
